catalog/app/containers/HomePage/index.js

```diff
@@ -29,9 +29,9 @@ const Styler = styled.div`
 export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
   render() {
     const { team = {} } = config;
-    const header = team.name ? team.name : undefined;
-    const tagline = team.name ? 'Team data catalog' : undefined;
-    const signUp = !team.name;
+    const header = team.id ? team.id : undefined;
```

Comment:
So just `team` now?

Reply:
this is a special case because we set a default value with destructuring above. so `team` is already `{}`.